src/pages/docs/matchmaker/concepts/lobby-tags.mdx

```diff
@@ -0,0 +1,61 @@
+# Lobby tags
+
+Lobby tags are an optional property that can be passed to the [`lobbies.find`](/docs/matchmaker/api/lobbies/find), [`lobbies.join`](/docs/matchmaker/api/lobbies/join), and [`lobbies.create`](/docs/matchmaker/api/lobbies/create) endpoints to allow filtering which lobbies the matchmaker will route a player to.
+
+To enable tags, set [`taggable`](http://localhost:3000/docs/general/config#matchmaker-game-modes-game-mode-taggable) to true.
+
+When passed to [`lobbies.create`](/docs/matchmaker/api/lobbies/create), the newly created lobby will have the given tags.
+
+<Warning>Tags are immutable. After a lobby is created, its tags cannot be changed.</Warning>
+
+## Example
+
+When passed to [`lobbies.find`](/docs/matchmaker/api/lobbies/find) or [`lobbies.join`](/docs/matchmaker/api/lobbies/join), the matchmaker will only return lobbies that have a **superset** of the given tags.
+
+Imagine this scenario where two lobbies were created with the tags and one without:
+
+1. **Lobby 1**
+
+   - `worldId = 123abc`
+   - `map = sandstorm`
+
+2. **Lobby 2**
+
+   - `worldId = foobar`
+   - `map = sandstorm`
+
+3. **Lobby 3**
+   - No tags
+
+Here is what would happen with the following [`lobbies.find`](/docs/matchmaker/api/lobbies/find) requests:
+
+1. **A [`lobbies.find`](/docs/matchmaker/api/lobbies/find) request with no tags**
+
+   The matchmaker would connect the player to any of the 3 lobbies.
+
+2. **A [`lobbies.find`](/docs/matchmaker/api/lobbies/find) request with the tags `map = sandstorm`**
+
+   The matchmaker would connect the player to either lobby 1 or lobby 2.
+
+3. **A [`lobbies.find`](/docs/matchmaker/api/lobbies/find) request with the tags `map = sandstorm`, `worldId = 123abc`**
+
+   The matchmaker would only connect players to lobby 1.
+
+4. **A [`lobbies.find`](/docs/matchmaker/api/lobbies/find) request with the tags `map = sandstorm`, `worldId = foobar`, `difficulty = hard`**
+
+   The matchmaker would error because there are no lobbies with the given tags. Note that there is a lobby with two of the requested tags, but it contains a **subset** of the tags in the request, not a **superset**.
+
+5. **A [`lobbies.find`](/docs/matchmaker/api/lobbies/find) request with the tags `map = tropic`**
+
+   The matchmaker would error because there are no lobbies with the given tags.
+
+<Note>
+  This is assuming
+  [`prevent_auto_create_lobby`](http://localhost:3000/docs/matchmaker/api/lobbies[`lobbies.find`](/docs/matchmaker/api/lobbies/find)#prevent-auto-create-lobby)
+  is `true`. If auto creating lobbies is enabled, cases 4 and 5 would create a new lobby with the given tags
+  and cases 1-3 would create a new lobby with the given tags only if all available servers are full.
```

Comment:
Can we move this at the top? I think talking about "cases 4 and 5" adds more confusion and things to think about. Just something like "uses prevent auto create lobby to demonstrate how lobbies are selected"